### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Help" button in  plugins dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1456]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. Tab to Help (?) button and hit Space bar then observe keyboard focus .

**Expected Result:**
Keyboard focus  should visible on "Help" button in plugins dialog.

**Actual Result:**
Keyboard focus is not visible on "Help" button in plugins dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![keybaord focus is not visible on help button 2](https://user-images.githubusercontent.com/22956130/36012821-9a774baa-0d15-11e8-99b0-d806fe760ede.png)

**Comment by Eskibear** [comment]
Cannot reproduce now in version: 2017.3.4 Community

![image](https://user-images.githubusercontent.com/2351748/39558305-1d2c0c02-4ec0-11e8-94cc-aca0ecffe44b.png)

**Comment by Eskibear** [comment]
a related bug is submitted to IntellIJ, https://youtrack.jetbrains.com/issue/IDEA-191682

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. This is the feature of IntelliJ IDEA itself. It's out of the plugin scope.
* **Quality of experience**: not impacted. This cannot be controlled by the plugin since it is the IntelliJ IDEA's feature. But if the user do want to see the help information, he can view the online docs for IntelliJ. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution . Can you please remove A11YExclusion, wontfix, Resolved, notreproduce labels wherever applicable and add ExclusionAgreed

**Comment by WiproPSAccTest** [comment]
Please ignore the previous comment

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749884
